src/ImageHistory.jsx

```diff
@@ -12,7 +12,7 @@ const IdColumn = Id => {
     Id = utils.truncate_id(Id);
     // Not an id but <missing> or something else
     if (/<[a-z]+>/.test(Id)) {
-        return <div className="pf-v5-u-disabled-color-100">{Id}</div>;
+        return <div className="pf-v6-u-disabled-color-100">{Id}</div>;
```

Comment:
This should work AFAIK. Not tested but the disabled-color-100 doesn't seem to exist
```suggestion
        return <div className="pf-v6-u-background-color-disabled">{Id}</div>;
```

Reply:
That is easy to test, in the image overview, expand an image and click the History tab. With these changes the color is still not disabled